graylog2-plugin-interfaces/src/main/java/org/graylog2/plugin/inputs/transports/AbstractTcpTransport.java

```diff
@@ -200,6 +237,28 @@ public ConfigurationRequest getRequestedConfiguration() {
                             TextField.Attribute.IS_PASSWORD
                     )
             );
+            x.addField(
+                    new BooleanField(
+                            CK_TLS_NEED_CLIENT_AUTH,
+                            "TLS Need Client Auth",
+                            false,
+                            "TLS Need Client Auth")
```

Comment:
I think it would be good to have a small description what "Need Client Auth" (and also "Want Client Auth") actually means.